### PR TITLE
en: separate client certificates for each component

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Update document for separate client certificates for each component.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/275, https://github.com/pingcap/docs-tidb-operator/pull/321
- Other reference link(s):

### CODE REVIEW

File: en/enable-tls-for-mysql-client.md [hunk 1]
```diff
@@ -10,18 +10,18 @@ This document describes how to enable TLS for MySQL client of the TiDB cluster o
...
     - [Using the `cert-manager` system](#using-cert-manager)
 
-2. Deploy the cluster, and set `.spec.tidb.tlsClient.enabled` to `true`.
+2. [Deploy the cluster](#deploy-the-tidb-cluster), and set `.spec.tidb.tlsClient.enabled` to `true`.
 
-3. Configure the MySQL client to use encrypted connection.
+3. [Configure the MySQL client to use encrypted connection](#configure-the-mysql-client-to-use-encrypted-connection).
 
-## Step 1: Issue two sets of certificates for the TiDB cluster
+## Issue two sets of certificates for the TiDB cluster
```
[the remark this answers is not in the document]
Comment (ran-huang) [reply en/enable-tls-for-mysql-client.md:24]: A suggestion from Liquan Pei. Some of our docs don't use `step 1, 2,...` in heading, so I think we'd better unify the usage. What do you think?

[the remark this answers is not in the document]
Comment (TomShawn) [reply en/enable-tls-for-mysql-client.md:24]: Fine.

**Comment by ran-huang** [comment]
@TomShawn PTAL again. Thanks!

**Pull request merged**